include/sound/sof/header.h

```diff
@@ -117,6 +117,7 @@
 #define SOF_IPC_TRACE_DMA_PARAMS		SOF_CMD_TYPE(0x001)
 #define SOF_IPC_TRACE_DMA_POSITION		SOF_CMD_TYPE(0x002)
 #define SOF_IPC_TRACE_DMA_PARAMS_EXT		SOF_CMD_TYPE(0x003)
+#define SOF_IPC_TRACE_FILTER_UPDATE		SOF_CMD_TYPE(0x004)
```

Comment:
suggested edits on the commit message (please correct if I misunderstood)

The "dma_trace_filter" debugfs file defines the log levels used by the firmware and reported by sof-logger

The file format follows the following syntax in plain text:
log_level uuid_id pipe_id comp_id; <value missing?>

This file may be updated by userspace applications such sof-logger, or directly by the user during debugging process.

An unused pipe_id or comp_id value should be set to `-1`, 
uuid_id is hexadecimal value, so when unused then should be set to 0.

<does -1 mean wildcard?> 

When the file is modifed, an IPC command is sent to FW with new trace levels for selected components
in filter elements list.

Reply:
updated